presto-cli-provisio/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please add a comment what this module is supposed to do and why we need a different module apart from `presto-cli-rpm`

Reply:
It's a little long but [here](https://github.com/prestosql/presto/pull/4025/files#diff-4ba198ba64f21a442f8a95c010e07ffbR2-R6).

Reply:
Could you make this part of `pom.xml`, so that future code readers can get the idea?

Reply:
The root pom.xml? Or this pom? I've added it to `presto-cli-provisio/pom.xml`.